b2share/version.py

```diff
@@ -21,4 +21,4 @@
 """Version number."""
 
 
-__version__ = "2.0.0"
+__version__ = "2.0.1.dev20170307"
```

Comment:
This is the `.dev` part [as described here](https://www.python.org/dev/peps/pep-0440/#public-version-identifiers)